packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -15,22 +15,39 @@ const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
 exports.upload = function upload(id, payload, headers) {
-  return new P(function (resolve, reject) {
+  return new P(function(resolve, reject) {
     var url = WORKER_URL + '/a/' + hex(id);
 
+    const header_excludes = config.get('worker.headers_exclude');
+
+    let worker_headers = {};
+    if (header_excludes.length) {
+      // pass-through all headers except these
+      for (const header in headers) {
+        if (header_excludes.includes(header.toLowerCase())) {
```

Comment:
style nit: I suppose we could save a line here if we check for `if (!header_excludes.includes(header.toLowerCase()))` and then copy the header inside the if block.

Reply:
OK, did some thinking and actually was able to removed some more lines with the new version.